.circleci/config.yml

```diff
@@ -53,7 +53,7 @@ commands:
         default: 6
     steps:
       - browser-tools/install-firefox:
-          version: 78.9.0esr
+          version: 91.5.0esr
```

Comment:
Interesting! According to https://whattrainisitnow.com/, the current ESR is 102.4.0.
I can't find a decent ESR calendar/roadmap, apart from trolling the https://ftp.mozilla.org/pub/firefox/releases/ directory here (which says [91.9.1esr/](https://ftp.mozilla.org/pub/firefox/releases/91.9.1esr/) might be the latest 91.x ESR build).

Reply:
Upgraded to ESR 102.4.0 and Geckodriver 0.32.0